### ISSUE TITLE
Neural Forecast: Tweedie Distribution Loss Broken (NBEATSx)

### ISSUE THREAD

**Issue by jrodenberg0** [Nixtla/neuralforecast#1181]
### What happened + What you expected to happen

Attempting to train NBEATsx with NeuralForecast and Tweedie loss and encounter error claiming that "an output must be a tensor, not a tuple") 

### Versions / Dependencies

neuralforecast                     1.7.5

pytorch-forecasting                1.1.1
pytorch-lightning                  2.4.0

### Reproducible example

from neuralforecast import NeuralForecast

from neuralforecast.models import NHITS, NBEATSx

from neuralforecast.losses.pytorch import DistributionLoss,Tweedie, MSE,MAE,RMSE

import torch

horizon = 13

rho_ = torch.tensor([1.5])#.to('cuda:0') # attempted to take rho to tensor to solve bug, did not work.

cuda = torch.device('cuda')

rho_ = rho.to(cuda)

tweedie_loss = DistributionLoss(distribution='Tweedie',rho=1.2,validate_args=False,return_params=True)

#define models here
models = [NBEATSx(h=horizon,

                  input_size = 52,

                  #max_steps=200,

                 # scaler_type='robust',

                  loss= DistributionLoss(distribution='Tweedie',rho=1.2,validate_args=False)
,
                 
                  activation='ReLU',

                val_check_steps=10,

                  stat_exog_list = ['Client','Warehouse','Product'],
                  
                  enable_model_summary=False,
            
                  enable_checkpointing=False,
            
                  logger=False

                  #date_features=['week','month']
                  ),

      # NHITS(h=horizon,

      #       loss=MAE(),

      #       input_size=104,

      #             stat_exog_list = ['Client','Warehouse','Product'],

      #       )
               ]

nf = NeuralForecast(
    models=models,
    freq='W-MON'
)

nf_df = df.select('unique_id','Client','Warehouse','Product','ds','y').to_pandas()

nf_df['y'] = nf_df['y'].fillna(0)

dl_cv = nf.cross_validation(df=nf_df,

                id_col='unique_id',

                time_col='ds',

                     n_windows=5,

                     step_size=horizon,

                            static_df = nf_df[['unique_id','Client','Warehouse','Product']])

### Issue Severity

High: It blocks me from completing my task.
